### ISSUE TITLE
Share some files with others

### ISSUE THREAD

**Issue by eikek** [eikek/docspell#446]
Just an idea – It would be nice to allow to share certain files with some one outside of docspell, for example with your tax advisor. Docspell could generate unique URLs that can be shared that would show only a selected subset of items.

**Comment by totti4ever** [comment]
Most probably it would that way anyway, but it would help a lot if the path for shared file was unique, i.e. docspell.tld/shared or /s or whatsoever

Why? That way one could whitelist only those urls for puclic access in a reverse proxy like _traefik_

**Comment by eikek** [comment]
Yes, this will be on a different path, because it will be a different page. Very similar to the public source page that exists, like `/app/upload/5JEYQa3SfQ…`. I also thought about protecting it with a password …. essentially duplicating the functionality from sharry ;-). But here it would be possible to allow searching in only a subset of items.
